docs/configure

```diff
@@ -77,8 +77,9 @@ ALIAS_DOMAINS = SELECT DISTINCT alias FROM domainalias WHERE alias = '${quote_my
 domainlist local_domains = @ : example.org : ${lookup mysql{VIRTUAL_DOMAINS}} : ${lookup mysql{ALIAS_DOMAINS}}
 domainlist relay_to_domains = ${lookup mysql{RELAY_DOMAINS}}
 hostlist   relay_from_hosts = localhost : @ : 192.168.0.0/24
-trusted_users = avleen:www
+trusted_users = vexim:www-data
```

Comment:
Well again, no process runs under vexim user, so by extension, no process running under that user would send any e-mails. I agree we can document this somewhere. Perhaps in the release notes?